harness/determined/common/api/profiler.py

```diff
@@ -1,7 +1,7 @@
+import time
 from typing import Any, Dict, List, Optional
 
-import backoff
```

Comment:
why not remove the other two instances of `@backoff` and just drop the dependency altogether?